test/fixtures/imminence/london.json

```diff
@@ -0,0 +1,69 @@
+{

```

Comment:
I believe this file can be deleted once you switch to using `imminence_has_areas_for_postcode`.